autopilot/sagemaker_autopilot_neo4j_portfolio_churn.ipynb

```diff
@@ -0,0 +1,916 @@
+{
```

Comment:
<p>You don't need to worry about cleaning up S3 files - just make sure to delete any resources that keep incurring cost. Does the customer need to delete the ASGs associated with Neo4j? </p><p> </p><p>Optionally add a Conclusion summarizing what the customer accomplished, and suggest any next steps, perhaps deploying the model to an endpoint, or further SageMaker ML applications using Neo4j.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>